### ISSUE TITLE
Every device can open shaders

### ISSUE THREAD

**Issue by Ilkerkemec99** [ZalithLauncher/ZalithLauncher#308]
### Feature Description / 功能描述

I can't open shaders on my own device and to fix this I would like to have a feature that allows us to change the mesa version as we want so that we can open shaders on most devices. My device's vulkan version is old so shaders can't be opened. I saw this feature on another browser and I would be very happy if it was on this one too. Please bring this feature.
![Screenshot_2024-12-05-02-05-18-493_com movtery zalithlauncher](https://github.com/user-attachments/assets/191d3a39-3e39-44c2-828c-f7795b4f89ab)
![Screenshot_2024-12-05-02-05-58-529_net kdt pojavlaunch firefly](https://github.com/user-attachments/assets/a89f192e-efb0-4226-955f-4ff377c55e66)

### Additional Information / 其他信息

So this feature increases your desire to use this launcher, please try to do this.

### This feature request has not been proposed before / 这个功能建议还没有人提出过

- [X] I have checked the Issues and found no similar feature request / 我已经查看过 Issues，并且没有发现与之类似的功能建议被提出

**Comment by azizir5** [comment]
I want this feature for running shaders that recommande new version of mesa and i want to run zink in opengl 4.6

**Comment by ilkerpro78** [comment]
> Mesa'nın yeni sürümünü öneren shader'ları çalıştırmak için bu özelliği istiyorum ve OpenGL 4.6'da Zink'i çalıştırmak istiyorum

Yes, we should announce this request to the producers. If they do, you will be able to run ZINK even if your device does not support it.
